vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/AuthenticationRequestParametersFrom.kt

```diff
@@ -0,0 +1,16 @@
+package at.asitplus.wallet.lib.oidc
+
+import io.ktor.http.Url
+
+sealed class AuthenticationRequestParametersFrom<T>(val source: T, val parameters: AuthenticationRequestParameters) {
```

Comment:
But isn't this now the same as the other class `AuthenticationRequest`?

Reply:
Yes, the goal of AuthenticationRequest was to have this class, but serializable